File: abc163/c/c.ts
import * as fs from "fs";

function manage(n: number, a: number[]): number[] {
  const res: number[] = new Array(n + 1).fill(0);
  for (const i of a) {
    res[i] += 1;
  }
  return res;
}

function main() {
  const input = fs.readFileSync("/dev/stdin", "utf-8").trim().split(/\s+/).map(Number);
  const n = input[0];
  const a = input.slice(1);
  const ans = manage(n, a)

  console.log(ans.slice(1).join("\n"));
}

main();

File: abc163/c/c.py
def manage(n, a):
    res = [0] * (n + 1)
    for i in a:
        res[i] += 1
    return res


def main():
    n = int(input())
    a = list(map(int, input().split()))
    ans = manage(n, a)
    for i in range(1, n + 1):
        print(ans[i])


if __name__ == "__main__":
    main()
